src/components/Typography/Typography.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Typography as TypographyMUI, TypographyProps } from '@mui/material';
+import '../../config/fonts/styles.css';
+import {
+  ThemeProvider,
+  responsiveFontSizes,
+  createTheme,
+} from '@mui/material/styles';
+
+export const Typography = (props: TypographyProps) => {
+  const {
+    children,
+    variant,
+    align,
+    gutterBottom,
+    noWrap,
+    paragraph,
+    fontFamily,
+  } = props;
+
+  let theme = createTheme();
+  theme = responsiveFontSizes(theme);
```

Comment:
Does this work for you? It doesn't seem to change the font size for me.

Reply:
That is a really good catch! Was working for me, but need to double check, because was struggling with it and most probably removed some code.

Reply:
As we spoke, added a fix for that